website/src/try.njk

```diff
@@ -19,6 +19,13 @@ permalink: "/try.html"
 		<div class="spinner"></div>
 	</div>
 	<div style="display: none" data-bind="visible: true">
+		<!-- ko if: hasUnsavedChanges -->
+			<div class="text-right">
```

Comment:
When page is cacheable, regular input fields are restored when user navigates back or reopen that tab. Forced refresh reload the page fully. I would probably prefer this behaviour.

Also https://github.com/phpstan/phpstan/issues/11838 should be fixed first, as currently the previous changes are lost even if they have permanent url. (lost in sense user cannot navigate back if he did not save the url)

Reply:
> When page is cacheable, regular input fields are restored when user navigates back or reopen that tab. Forced refresh reload the page fully. I would probably prefer this behaviour.

Not sure what you really mean here...


> Also https://github.com/phpstan/phpstan/issues/11838 should be fixed first, as currently the previous changes are lost even if they have permanent url. (lost in sense user cannot navigate back if he did not save the url)

Nothing says that #11838 has to be fixed first---that's orthogonal to this and can be fixed independently